docker-image/Dockerfile

```diff
@@ -0,0 +1,23 @@
+FROM buildpack-deps:bullseye
+
+RUN set -eux; \
+	apt-get update; \
+	apt-get install -y --no-install-recommends \
+        wget \
+	; \
+	rm -rf /var/lib/apt/lists/*
+
+ENV CONDA_DIR /opt/conda
+
+RUN wget https://repo.anaconda.com/miniconda/Miniconda3-latest-Linux-x86_64.sh -O ~/miniconda.sh && \
+     /bin/bash ~/miniconda.sh -b -p /opt/conda
+
+ENV PATH=$CONDA_DIR/bin:$PATH
+RUN conda install -y -c anaconda python=3.8 \
```

Comment:
Why do you downgrade Python to 3.8 in the base environment?

Reply:
Because appeared to me that you were using 3.8, it was one of the trickiest thing to do.